PowerShell Scanners/Neighbor Cache Information/Neighbor Cache Information.ps1

```diff
@@ -0,0 +1,97 @@
+[CmdletBinding()]
+param(
+	# Define the maximum number of runspaces. Default value is the number of processor son the machine + 1
+	[int]$MaxRunspaces = ( [int]$env:NUMBER_OF_PROCESSORS + 1),
```

Comment:
I think this can be up'd a good amount. For giggles I ran this at 400 threads and there was little in the way of impact on my laptop. Hyper-V running WSL2 was still the most resource intensive.

I think something like `[int]$env:NUMBER_OF_PROCESSORS * 25` would be safe (maybe a little too safe, my gut says `* 100` would be better but I have nothing to back that up) but it's hard to say without results of this from our users on what the right scaling here is.

Reply:
Updated to 25